webpack/scenes/Subscriptions/SubscriptionReducer.js

```diff
@@ -91,7 +89,9 @@ export default (state = initialState, action) => {
     }
 
     case DELETE_SUBSCRIPTIONS_SUCCESS:
-      return state.set('loading', false);
+      return state
+        .set('deleteButtonDisabled', true)
```

Comment:
In addition to disabling the button we need to clear whatever is keeping track of the selection.

After I delete 1 subscription and it completes, I select another 1, the modal says this:

```
Are you sure you want to delete 2 subscription(s)? This action will remove the subscription(s) and refresh your manifest. All systems using these subscription(s) will lose them and also may lose access to updates and Errata.
```

It should say 1 subscription

Reply:
@sseelam2 have you had a chance to look into this?

Reply:
@jturel I am working on it. I am looking into how to change the state of `selectedRows` because that is keeping track of the selection and it is not getting updated once the delete action is complete.

Reply:
Used refs to change the state of the child component from its parent!

Reply:
@sseelam2 I have to give you props for your unorthodox methods, which I definitely would not have thought of.  But look at what the React folks [say here](https://reactjs.org/docs/refs-and-the-dom.html#when-to-use-refs):

> Your first inclination may be to use refs to “make things happen” in your app. If this is the case, take a moment and think more critically about where state should be owned in the component hierarchy. Often, it becomes clear that the proper place to “own” that state is at a higher level in the hierarchy. See the [Lifting State Up](https://reactjs.org/docs/lifting-state-up.html) guide for examples of this.

I'd have to hear a very compelling argument for not using state or Redux to solve this.

Reply:
Ah. I thought it was safe to use refs. Updated the PR. Lifted the state up :+1: